src/package_io/op_input_parser.star

```diff
@@ -102,7 +102,7 @@ def _default_args(log_format=constants.LOG_FORMAT.json):
 
 DEFAULT_NON_NATIVE_ARGS = _sort_dict_by_values(
     {
-        "source": "github.com/agglayer/optimism-package/main.star@2769472af9802d5e0f460ce27ebba32de1d21005",  # overlay/main - 2025-10-18
+        "source": "github.com/agglayer/optimism-package/main.star@66b5f81ab2034b4592e4c963a5f4c858fc7e6e58",  # fix/op-node-l1-genesis - 2025-11-14
```

Comment:
update this commit hash once `fix/op-node-l1-genesis` branch is merged into `main`

Reply:
done in [6250031](https://github.com/0xPolygon/kurtosis-cdk/pull/823/commits/6250031b6aa1d254e152fcd90c6eaa823e777715)